optuna_dashboard/ts/components/CompareStudies.tsx

```diff
@@ -321,19 +322,21 @@ const StudiesGraph: FC<{ studies: StudySummary[] }> = ({ studies }) => {
           </CardContent>
         </Card>
       ) : null}
-      {showStudyDetails !== null &&
-      showStudyDetails.length > 0 &&
-      showStudyDetails.every((s) => s) ? (
-        <Card
-          sx={{
-            margin: theme.spacing(2),
-          }}
-        >
-          <CardContent>
-            <GraphEdfMultiStudies studies={showStudyDetails} />
-          </CardContent>
-        </Card>
-      ) : null}
+      <Grid2 container spacing={2} sx={{ padding: theme.spacing(0, 2) }}>
```

Comment:
Should we move this into L329? Otherwise, I think Optuna Dashboard may render an empty `<Grid2></Grid2>` tag.